src/relay/transforms/fuse_ops.cc

```diff
@@ -247,6 +247,28 @@ class IndexedForwardGraph::Creator : private ExprVisitor {
       this->Update(call->op, node, kOpaque);
     }
 
+    if (call->attrs.as<StridedSliceAttrs>()) {
+      bool is_dyn{false};
+      for (auto arg :  call->args) {
+        if (!arg.as<ConstantNode>()) {
+           is_dyn = true;
+           break;
+        }
+        auto arg_tt = arg->checked_type().as<TensorTypeNode>();
+        if (arg_tt) {
```

Comment:
Since we have already checked begin, end and stride through as<ConstantNode>, I think here we just need to check the first arg?